src/main.rs

```diff
@@ -33,9 +35,20 @@ Options:
   -n, --bench     Run `cargo bench` when a file is modified
 
 Default options are `build` and `test`
-");
+";
 
-#[derive(Copy)]
+#[derive(RustcDecodable, Debug)]
+struct Args {
+      flag_build: bool,
+      flag_doc: bool,
+      flag_test: bool,
+      flag_bench: bool,
+    //flag_help: _, // These three flags are not used. They are generated by docopt
```

Comment:
Are those three flags really not needed anymore to support both `cargo watch` and `cargo-watch` invocations? I didn't write the original docopt code, so I don't really know, but can you please confirm this?